metashare/api/gh.py

```diff
@@ -0,0 +1,47 @@
+"""
+GitHub utilities
+"""
+
+from django.core.exceptions import (
+    MultipleObjectsReturned,
+    ObjectDoesNotExist,
+    ValidationError,
+)
+from github3 import login
+from purl import URL
+
+
+def get_all_org_repos(user):
+    try:
+        token = (
+            user.socialaccount_set.get(provider="github").socialtoken_set.get().token
+        )
+    except (ObjectDoesNotExist, MultipleObjectsReturned):
+        return set()
+    gh = login(token=token)
+    repos = set(
+        [
+            normalize_github_url(repo.url)
+            for org in gh.organizations()
+            for repo in org.repositories()
+        ]
+        + [normalize_github_url(repo.url) for repo in gh.repositories()]
```

Comment:
@wlonk This is still only showing me user repos, not org repos. Might take a bit more digging?